common/messages/dbstateMissing_test.go

```diff
@@ -81,35 +81,36 @@ func newDBStateMissing() *DBStateMissing {
 	return msg
 }
 
-func Testlimits(t *testing.T) {
-
-	for s := uint32(0); s < 100; s++ {
-		for e := s; e < s+1000; e++ {
-			for in := 0; in < 1000; in++ {
-				ns, ne := NewEnd(in, s, e)
-
-				if s != ns {
-					t.Errorf(" Failed with e %d s %d and in %d", e, s, in)
-				}
-
-				if ne > e {
-					t.Errorf(" Failed with e %d s %d and in %d", e, s, in)
-				}
-
-				if in > 500 && ne != 0 {
-					t.Errorf(" Failed with e %d s %d and in %d", e, s, in)
-				}
-
-				if in <= 500 && in > 200 && ne-ns > 50 {
-					t.Errorf(" Failed with e %d s %d and in %d", e, s, in)
-				}
-
-				if in <= 200 && ne-ns > 200 {
-					t.Errorf(" Failed with e %d s %d and in %d", e, s, in)
-				}
-			}
-		}
-	}
+func TestLimits(t *testing.T) {
```

Comment:
this test used to not run because of the faulty name. After being enabled it failed because of timeouts and/or out of memory. This test doesn't make any sense to test such a trivial function such as NewEnd